SkeletonableTableView/Sources/SkeletonableTableView/SkeletonableTableView.swift

```diff
@@ -0,0 +1,188 @@
+//
+//  SkeletonableTableView.swift
+//  SkeletonableTableView
+//
+//  Created by Almas Zainoldin on 7/15/20.
+//  Copyright © 2020 crococoders. All rights reserved.
+//
+
+import SkeletonView
+import UIKit
+
+private enum Constants {
+    static let cornerRadius = 4
+}
+
+public final class SkeletonableTableView: UITableView {
```

Comment:
This one work if people use it as-is without inheriting their own base tableview implementations, but in case if they do - they'd need to remove the `final` keyword. I think you can just do it for them